### PR TITLE
bluetooth: controller: Enable new feature supports

### PR DESCRIPTION
**Pull request by ilhanates**
Peripheral initiated feature exchange support is added to sdc.
The feature is supported as default.

Signed-off-by: Ilhan Ates <ilhan.ates@nordicsemi.no>

### CODE REVIEW

**Comment by ilhanates** [comment]
@seko-nordic We will need to add nrfxlib PR that will bring the feature support to ncs. I can update nrf west manifest here to point to the nrfxlib PR when it is ready (tested & qualified).

**Review by rugeGerritsen — APPROVED** [review]

**Comment by ilhanates** [comment]
Some ble tests are failing due to LL procedure collisions after peripheral initiated feature exchange is enabled in this PR. This possibly requires SDC updates. I will be following further.

**Comment by ilhanates** [comment]
I removed manifest update as the related feature are already merged in ncs by other recent nrfxlib updates

File: subsys/bluetooth/controller/hci_internal.c [hunk 1]
```diff
@@ -405,6 +405,7 @@ static void le_supported_features(sdc_hci_le_le_features_t *features)
 
 	features->le_encryption = 1;
 	features->extended_reject_indication = 1;
+	features->slave_initiated_features_exchange = 1;
```
Comment (rugeGerritsen) [inline subsys/bluetooth/controller/hci_internal.c:408]:
```suggestion
#if CONFIG_BT_PERIPHERAL
	features->slave_initiated_features_exchange = 1;
#endif
```

Comment (ilhanates) [reply subsys/bluetooth/controller/hci_internal.c:408]: These are controller supported features, not configured per role right?

Comment (seko-nordic) [reply subsys/bluetooth/controller/hci_internal.c:410]: both sides must support the feature for it to be usable. I.e. master also must declare that it is able to respond to it, right?

Comment (rugeGerritsen) [reply subsys/bluetooth/controller/hci_internal.c:410]: Ah, yes, @seko-nordic , you are right. This change should be reverted.

**Review by seko-nordic — APPROVED** [review]

**Pull request merged**